CMakeLists.txt

```diff
@@ -72,7 +78,7 @@ if (NOT "${ARCH}" STREQUAL "")
   endif()
 endif()
 
-if(WIN32 OR ARM7 OR ARM6)
+if(WIN32 OR ARM7 OR ARM6) #ARM8 may benefit from vectorisation
   set(OPT_FLAGS_RELEASE "-O2")
```

Comment:
I assume the decision to special-case Windows32 and ARM to -O2 and let further optimizations on others was conservatism because it should work with -Ofast. Ideally, there would be no special-case, and all would be built with -Ofast. If unit tests pass and node works on ARM8 with -Ofast, then agreed on pulling at least ARM8 out of the special case.